.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        scala: [2.13.4, 2.12.12]
+        scala: [2.13.3, 2.12.12]
```

Comment:
awwwww.... really?

Reply:
There's a lot tied up in this and I have time to dive into scalafix and see why it's generating invalid code.

Reply:
I kept the changes that make it compile on Scala 2.13.4.  Hopefully we can bring it back soon.

Reply:
ah, the Scalafix thing. yeah, fair.